Firmware/firmwareLiBCM/config.h

```diff
@@ -145,9 +145,14 @@
         //#define LED_DEBUG //enable "debugLED()" functions (FYI: blinkLED functions won't work)
 
     //#define LIDISPLAY_DEBUG_ENABLED //uncomment to enable updates to text box ID # T12 on LiDisplay driving page -- this shows raw comm data from LiDisplay to LiBCM
-    #define LIDISPLAY_CELL_COLOR_BIN_SIZE_COUNTS 64 //64 = 6.4mV window between cell colours on the grid charging page.  Don't go below CELL_BALANCE_TO_WITHIN_COUNTS_LOOSE
-	#define LIDISPLAY_SPLASH_PAGE_MS 3000 // How long the splash page shows on LiDisplay.  Default 3000 (3 seconds)
-	#define LIDISPLAY_GRID_CHARGE_PAGE_COOLDOWN_MS 4000 // Keep displaying the grid charging page this long before showing splash page when GC unplugged
+    #define LIDISPLAY_CELL_COLOR_BIN_SIZE_COUNTS 64 // 64 = 6.4mV window between cell colours on the grid charging page.  Don't go below CELL_BALANCE_TO_WITHIN_COUNTS_LOOSE
+	#define LIDISPLAY_SPLASH_PAGE_MS 5000 // How long the splash page shows on LiDisplay.  Default 3000 (3 seconds)
+	#define LIDISPLAY_GRID_CHARGE_PAGE_COOLDOWN_MS 5000 // Keep displaying the grid charging page this long before showing splash page when GC unplugged
+	#define PERIOD_TO_DISABLE_SLEEP_AFTER_KEYOFF_MS 8000 // Default is the sum of LIDISPLAY_GRID_CHARGE_PAGE_COOLDOWN_MS and PERIOD_TO_DISABLE_SLEEP_AFTER_KEYOFF_MS
+	#define LIDISPLAY_SPLASH_PIC_ID 21 // Default is 21 for the old Citrus photo.  New options are Citrus 59, Silver 60, and Red 61
```

Comment:
Make this a define select.  Put these in liDisplay.h:
#define COLOR_CITRUS 21
#define COLOR_RED 61
#define COLOR_SILVER 60

Put these here in config.h
#define LIDISPLAY_SPLASH_PIC COLOR_CITRUS
//#define LIDISPLAY_SPLASH_PIC COLOR_RED
//#define LIDISPLAY_SPLASH_PIC COLOR_SILVER